docs/source/quicktour.mdx

```diff
@@ -10,26 +10,41 @@ an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express o
 specific language governing permissions and limitations under the License.
 -->
 
-# Quick tour
+# Quicktour
 
-Let's have a look at 🤗 PEFT's main features and learn how to set up a `PeftModel` and train it with 🤗 Accelerate's DeepSpeed integration and use it for inference.
+🤗 PEFT contains parameter-efficient fine-tuning methods for training large pretrained models. The traditional paradigm is to fully train a model for each downstream task, but this is becoming exceedingly costly and impractical because of the enormous number of parameters in models today. Instead, it is more efficient to train a smaller number of prompt parameters or use a reparametrization method like low-rank adaptation (LoRA) to reduce the number of trainable parameters. 
 
-## Main use
+This quicktour will show you 🤗 PEFT's main features and help you train large pretrained models that would typically be inaccessible on consumer devices. You'll see how to train the 1.2B parameter [`bigscience/mt0-large`](https://huggingface.co/bigscience/mt0-large) model with LoRA to generate a classification label and use it for inference, and also how to train a 🤗 PEFT model with 🤗 Accelerate's DeepSpeed integration on distributed setups.
 
-To use 🤗 PEFT in your script:
+## PeftConfig
 
-1. Each PEFT method is defined by a `PeftConfig` object.
+Each 🤗 PEFT method is defined by a [`PeftConfig`] class that stores all the important parameters for configuring the model. 
```

Comment:
It can be unclear here what model is configured. Maybe, "Each 🤗 PEFT method is defined by a [`PeftConfig`] class that stores all the important parameters for building a parameter-efficient wrapper model." ?